lib/token.cpp

```diff
@@ -45,6 +45,13 @@
 #include <unordered_set>
 #include <utility>
 
+struct less {
```

Comment:
I suggest that we put less in a anonymous namespace

Reply:
Will do.

Reply:
Done.